docs/PROMETHEUS_METRICS.md

```diff
@@ -0,0 +1,136 @@
+# Prometheus-Compatible Metrics in EVE
+
+This document describes how EVE exposes system metrics in the Prometheus exposition format.
+
+## Node Exporter Metrics
+
+EVE exposes a broad set of metrics under the `node_exporter` style.
+From the EVE itself, you can access these metrics at:
+
+```text
+https://localhost:9100/metrics
+```
+
+### Integration into EVE
+
+`node_exporter` is defined as a service in `rootfs.yml.in`. The container
+runs with host network and PID namespaces to access system-wide process and
+networking information. Key system directories are mounted read-only under
+`/hostfs` to allow safe metric collection without modifying the host.
+
+The `Dockerfile` for the service is located in `pkg/node-exporter`, and its
+runtime config is defined in `build.yml`. The image version is pinned via
+`NODE_EXPORTER_TAG` in the build system (`tools/parse-pkgs.sh`), ensuring
+consistent deployments.
+
+To modify how metrics are collected or adjust the exporter’s flags, edit the
+`CMD` in the `Dockerfile` or update bind mounts in build.yml.
+
+### Exposed Metrics Categories
```

Comment:
Are these names and units part of some de-facto standard?

I'm trying to understand whether we are inventing some new schema for this or leveraging what everybody already does. 
For instance, can we replace/rename/retire metrics over time or are we committing this like an API (where additions are ok, but replace/rename/retire are not unless we move the API to a new major version.)

Reply:
No, we are not inventing anything new. The metrics, their names, and units follow the standard exposed by the node_exporter tool (https://github.com/prometheus/node_exporter). We simply redirect these metrics transparently. This also means that the API is not under our explicit control - whatever node_exporter sends, we send. As a result, while we can add our own additional metrics if needed, replacing and renaming existing ones should not be our task.